lib/platform/utils/pr-body.ts

```diff
@@ -0,0 +1,22 @@
+import { appName } from '../../config/app-strings';
+
+export function smartTruncate(input: string, len = 60000): string {
```

Comment:
I added 60000 to GitHub so I think it's best if we remove the default here

Reply:
done